src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -3534,23 +3792,26 @@ def test_sum_X(tmpdir, dist_mem):
 
 
 def test_sign_X(tmpdir, monkeypatch, annexed, dist_mem, fortran_writer):
-    ''' Test that 1) the str method of LFRicSignXKern returns the
+    '''Test that 1) the '__str__' method of 'LFRicSignXKern' returns the
     expected string and 2) we generate correct code for the built-in
-    operation Y = sign(a, X) where 'a' is a real scalar and Y and X
-    are real-valued fields. Test with and without annexed dofs
-    being computed as this affects the generated code.
+    operation 'Y = sign(a, X)' where 'a' is a real scalar and 'Y' and
+    'X' are real-valued fields. Test with and without annexed DoFs
+    being computed as this affects the generated code. Also test the
```

Comment:
I would suggest "3) Also test ..."

Reply:
Oops. Found quite a few of these in the file. Also noticed I sometimes say `Also tests`, rather than `Also test` so I changed these too.